todo_app/templates/index.html

```diff
@@ -11,8 +11,31 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+        {% for item in getlist%}
+		<li> {{item.id}} - {{item.title}} - {{item.status}} 
+      {% if item.status == "Not Started"%}
+      <form method="POST" action = "/complete">
+        <input type="hidden" name="itemcomplete" value = "{{item.id}}"> 
```

Comment:
Would be a bit more meaningful to have something like `name="item_id"` so it's clear what the data is when you're using it.